configs/workflows-mt.yaml

```diff
@@ -0,0 +1,377 @@
+Workflows:
+  - Name: Sequencing Noninterleaved
+    Collection: omics_processing_set
+    Enabled: True
+    Filter Output Objects:
+    - Metagenome Raw Read 1
+    - Metagenome Raw Read 2
+
+  - Name: Sequencing Interleaved
+    Collection: omics_processing_set
+    Enabled: True
+    Filter Output Objects:
+    - Metagenome Raw Reads
+
+  - Name: Sequencing
```

Comment:
@scanon  do we need to keep this?

Reply:
Those could be dropped since that isn't the model any longer.  I left it in for backwards compatibility but we have moved past that.